pkg/devicemapper/devmapper.go

```diff
@@ -264,11 +264,6 @@ func UdevWait(cookie *uint) error {
 	return nil
 }
 
-// LogInitVerbose is an interface to initialize the verbose logger for the device mapper library.
-func LogInitVerbose(level int) {
```

Comment:
How do we know that there are no external users of devicemapper package which are not making use of LogInitVerbose.

I think that might have been the thought in my mind and left LogInitVerbose() behind.